client/src/components/ui/forms/document/CreateDocument.svelte

```diff
@@ -34,10 +37,14 @@
             await reloadMetrics();
             this.reset();
         } catch (err) {
+            if (err instanceof DeferredSnap) {
+                await deferredSnaps.update(snaps => {return upsert(snaps, { status: Status.Register, doc: id });});
+                topToastMessage.enqueue({ title: err.name, body: `${id} is deferred.` });
+            }

```

Comment:
Perhaps you forgot to `return`?
```suggestion
                topToastMessage.enqueue({ title: err.name, body: `${id} is deferred.` });
                return;
            }

```